crates/sui-indexer-alt/src/ingestion/mod.rs

```diff
@@ -31,7 +32,11 @@ pub struct IngestionService {
 pub struct IngestionConfig {
     /// Remote Store to fetch checkpoints from.
     #[arg(long)]
-    remote_store_url: Url,
+    remote_store_url: Option<Url>,
+
+    /// Path to the local ingestion directory.
+    #[arg(long)]
+    local_ingestion_path: Option<PathBuf>,
```

Comment:
Can use an arg group to ensure that exactly one of these flags is passed in, and communicate that in the help and CLI parsing messages:

```
    #[arg(long, required = true, group = "source")]
    remote_store_url: Option<Url>,

    /// Path to the local ingestion directory.
    #[arg(long, required = true, group = "source")]
    local_ingestion_path: Option<PathBuf>,
```

(Unfortunately doesn't help much in the validation step below).